sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/implementation/AppConfigurationStatusException.java

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.config.implementation;
+
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.http.HttpResponse;
+
+class AppConfigurationStatusException extends HttpResponseException {
+
+    AppConfigurationStatusException(final String message, final HttpResponse response, final Object value) {
```

Comment:
>AppConfigurationStatusException [](http://example.com/codeflow?start=4&length=31)

Is this new error type only for our consumption? Or is this exposed to customers too i.e., will customers ever see `AppConfigurationStatusException` exception?

Reply:
This is new. No it isn't exposed to customers.